packages/cerebral-website-tutorial/parts/05_update_state/src/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { render } from 'react-dom'
+import { Controller } from 'cerebral'
+import App from './components/App'
+import { Container } from 'cerebral/react'
+import Devtools from 'cerebral/devtools'
+import { set } from 'cerebral/operators'
```

Comment:
and as well... still cerebral-http-provider is not ready for npm i think... can you help me out there?